### DISCUSSION TITLE
'Image' does not contain a definition for 'GetPixelMemoryGroup'

### DISCUSSION THREAD

**Discussion by georgekosmidis** [SixLabors/ImageSharp#1336] [Q&A] [answered]
Hello all,
could someone tell me why the following gives me `'Image' does not contain a definition for 'GetPixelMemoryGroup'`:
```
using SixLabors.ImageSharp.Advanced;
//...
            using (Image loadedImage = Image.Load(imagePath))
            {
                loadedImage.GetPixelMemoryGroup()
               //...
```

What am I missing here? Seems like right out of the docs :)

**Comment by antonfirsov** [comment, marked as answer]
You need to specify the pixel type to access the pixel buffers. The `Image` base type is intended to be agnostic of the underlying representation, since we plan to decode images in their native format for better perf.

Use `Image.Load<Rgba32>(...)` (or other pixel type), that will load an `Image<TPixel>`, the extension method is defined over that type.

**Comment by georgekosmidis** [comment]
Thank you very much!
